### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by HarshaVardhanJ**
Updated README.md with information on how to use the Docker images, Dockerfiles, and Docker Compose files.

### CODE REVIEW

**Comment by HarshaVardhanJ** [comment]
I've added some very basic documentation. I'll add more information in due time. Meanwhile, if you have any questions, do let me know.

File: README.md [hunk 1]
```diff
@@ -29,6 +31,22 @@ Upcoming:
...
+- Make sure that your system has Docker installed.
+
+If you wish to test the image for yourself, you can run
+
+```
+docker run -d --name archivy -p 5000:5000 -v /path/to/host/dir:/usr/src/app/data harshavardhanj/archivy:latest
+```
+and you can access the service at `http://localhost:5000/` on your browser. 
+
+Also, if you wish to build the image on your own, there are `Dockerfile`s present in the repository.
```
Comment (Uzay-G) [inline README.md:45]: Could you specify the two different Dockerfiles here and when to use them? 

I'll merge right after.

Thanks :+1:

**Comment by Uzay-G** [comment]
This is great and so extensive! Amazing :tada:

**Comment by HarshaVardhanJ** [comment]
Thank you. I'm glad you liked it. Working on a separate README related to Docker. Will create a PR soon.

**Pull request merged**